### ISSUE TITLE
How to bind IP any？

### ISSUE THREAD

**Issue by winxp2002** [Eastrall/LiteNetwork#54]
Sometimes it is impossible to have a real IP address，how to set ip as  " new IPEndPoint(IPAddress.Any, port)"

**Comment by winxp2002** [comment]
when I declare host to  0.0.0.0 
in code  LiteNetworkHelpers.CreateIpEndPointAsync 
IPAddress[] addresses = await Dns.GetHostAddressesAsync("0.0.0.0").ConfigureAwait(false);
Throw an Exception
System.ArgumentException:“IPv4 address 0.0.0.0 and IPv6 address ::0 are unspecified addresses that cannot be used as a target address. Arg_ParamName_Name”
It not work well
Please fix this bug

**Comment by Eastrall** [comment]
Thanks for reporting that using the `0.0.0.0` in server's host doesn't work. I will provide a fix ASAP.

**Comment by Eastrall** [comment]
Thanks for reporting that using the 0.0.0.0 in server's host doesn't work. Could you try to specify an empty string to your server's host?
```csharp
var configuration = new LiteServerOptions()
{
    Host = "",
    ...
};
```
Let me know if this workaround works.

**Comment by winxp2002** [comment]
empty string ，return IP is the first IP address of all network cards，Also the only an IP，Not equal to Any IP

**Comment by Eastrall** [comment]
You are correct. I will provide an update of the library to support the `0.0.0.0` address ASAP.

**Comment by Eastrall** [comment]
Version 2.1.0 has been released in NuGet: https://www.nuget.org/packages/LiteNetwork/2.1.0
Check out the release note here: https://github.com/Eastrall/LiteNetwork/releases/tag/v2.1.0

Please let me known if it solves your issue.

**Comment by winxp2002** [comment]
thanks ，it solved

**Comment by hoshinokanade** [comment]
I hoped this for a couple of while. @winxp2002 highly appreciated for raising this out.
